### DISCUSSION TITLE
Bulk set location field to null

### DISCUSSION THREAD

**Discussion by suncruise** [netbox-community/netbox#9350] [Q&A]
I need to set the location option to null for hundreds of devices. The only way I seem to be able to change the location to null is to manually change update each device model. I have tried setting the option to null using an Ansible script but no changes are applied, if I set the value to an empty string in results in an error. Using the bulk change option in GUI doesn't allow the value to be set to null. Am I missing something? If not would this be a candidate for a feature request?

**Comment by jcollie** [comment]
The quickest solution would be to script something in Python using pynetbox and the REST API.

**Comment by jeremystretch** [comment]
> If not would this be a candidate for a feature request?

Yes!
